addons/web/static/tests/legacy/views/calendar_tests.js

```diff
@@ -3877,7 +3877,7 @@ QUnit.module('Views', {
         assert.containsOnce(calendar, '.o_cw_popover');
         popoverText = calendar.el.querySelector('.o_cw_popover')
             .textContent.replace(/\s{2,}/g, ' ').trim();
-        assert.strictEqual(popoverText, 'December 12, 2016 event 2 10:55 event 3 15:55');
+        assert.strictEqual(popoverText, 'December 12, 2016 10:55 event 2 15:55 event 3');
```

Comment:
I am not sure we should change this. This would impact things globally. Can't the PO accept that the start_date is displayed after the name?